template/js-template/README.md

```diff
@@ -0,0 +1,50 @@
+# hardhat-web3
+
+### Web3.js Hardhat JS Template
+Contains template using: Web3.js, Hardhat, JavaScript and NPM
+
+## Introduction
+
+This project demonstrates a basic Hardhat use case, focusing on testing a Token smart contract. It includes a sample contract, a test for that contract, and a script that deploys the contract.
+
+The project utilizes:
+- Hardhat as the development environment
+- Chai for assertions
+- web3.js for interacting with Ethereum
+
+## Prerequisites
+
+- [Node.js and npm](https://nodejs.org/en/download/) (latest version)
+- Basic understanding of JavaScript and Ethereum smart contracts
+
+## Getting Started
+
+Follow these steps to set up and run the project locally.
+
+### Installation
+
+ Install NPM packages:
+   
+   `npm install`
+
+### Compiling the Contract
+
+Compile the smart contract with:
+
+`npx hardhat compile`
```

Comment:
when running this command i get an error
```
➜  js-template git:(release/v0.1.0) ✗ npx hardhat compile
Error HH12: Trying to use a non-local installation of Hardhat, which is not supported.
Please install Hardhat locally using npm or Yarn, and try again.

For more info go to https://hardhat.org/HH12 or run Hardhat with --show-stack-traces
```

Reply:
before `npx hardhat compile`,  install dependencies with `yarn` or `npm i`

Reply:
worked. thanks

Reply:
> What is the reasoning for not running these commands as part of the init script?

so users are aware of steps usually required for building proj.